index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style/styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+    <title>Joseph Kausi'Portfoliot</title>
+</head>
+<body>
+    <header id="main-head" class="family">
+        <h1>WELCOME</h1>
```

Comment:
The font size of this welcome text seems to be too much. Kindly confirm and set it to the correct font-size. see the image below: 

![image](https://user-images.githubusercontent.com/84156714/205169044-400607aa-b637-41c7-bd0b-40379efc4656.png)